util.cc

```diff
@@ -258,6 +258,19 @@ xfopen(const char* fname, const char* type, const char* errtxt)
   return f;
 }
 
+/*
+ * OS-abstracting wrapper for getting Unicode environment variables.
+ */
+QString ugetenv(const char* env_var) {
```

Comment:
I'd have named this "xgetenv" for consistency with our other C-era wrappers, but rather than asking to change it, I'll add it to the rainy day list of things to move to gb::something or src/core/something.cc or other better fit.

Reply:
I considered this, but felt that "xfopen" was a bit more than a thin wrapper around "fopen", whereas "ufopen" (from PR #87) is a Unicode-supporting thin wrapper, hence the "ugetenv".